src/data/dialogmote/dialogmoteTypes.ts

```diff
@@ -1,6 +1,7 @@
 export enum MotedeltakerVarselType {
   AVLYST = "AVLYST",
   INNKALT = "INNKALT",
+  REFERAT = "REFERAT",
```

Comment:
Tror ikke dette er en gyldig Type fra backend, men det er du kanskje klar over allerede.

Reply:
Typen finnes jo nå, men selve referatet vil ikke (for isyfo-formål) ligge som en instans i varsellista til arbeidstakeren. Referatet ligger som en egen instansvariabel rett på dialogmote-dto'en.

Reply:
Ja og da finnes ikke typen på `MotedeltakerVarselType `.